x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -484,6 +484,22 @@ static RoleDescriptor kibanaSystem(String name) {
                 // Endpoint heartbeat. Kibana reads from these to determine metering/billing for
                 // endpoints.
                 RoleDescriptor.IndicesPrivileges.builder().indices(".logs-endpoint.heartbeat-*").privileges("read", "create_index").build(),
+                // Security Solution workflows insights. Kibana creates, manages, and uses these
+                // to provide users with insights on potential configuration improvements
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(".security-workflow-insights-*")
```

Comment:
```suggestion
                    .indices(".kibana-security-workflow-insights-*")
```
Elasticsearch has already defined many `.security-*` indices (e.g. `.security-7`, `.security-profile`, `.security-token`). Introducing `.security-workflow-insights-*` as part of the `.security-*` namespace might lead to unexpected behaviour, cause a confusion or even lead to an accidental exposure. We would like to keep `.security-*` prefix only for Elasticsearch internally managed indices.

Since this new system index is meant to be managed by Kibana only, how about naming it `.kibana-security-workflow-insights-*` instead?

Reply:
Good catch! However, `.kibana*` is reserved for Kibana’s own data store/Saved Objects indices (cc @rudolf). I believe we`d need to use a use-case/area/solution-specific prefix here instead.

Reply:
Thanks for the context, that makes sense to me. Updated to `.kibana-security-workflow-insights-* ` 👍

Reply:
> Has there been any discussions around introducing a new system index with the Elasticsearch team?

No, I think this is closer to an internal index rather than a system index. Similar to the other `.` prefixed indices installed by the Elastic Defend package.

Reply:
updated to `.edr-workflow-insights`